code/game/objects/items/weapons/shields.dm

```diff
@@ -48,6 +50,12 @@
 /obj/item/weapon/shield/proc/get_block_chance(mob/user, var/damage, atom/damage_source = null, mob/attacker = null)
 	return base_block_chance
 
+/obj/item/weapon/shield/attack(mob/M, mob/user)
+	if(isliving(M))
+		var/mob/living/L = M
+		L.slowdown += slowdown_time
```

Comment:
Does the max slowdown you can give by how much you can attack or that's infinite ?

Reply:
Yeah, you right. I will add limit to this one.

Reply:
Limit is slowdown_time * 3.

Reply:
Done.